migrations/v1_2_2_data_permissions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ *
+ * Advanced Polls
+ *
+ * @copyright (c) 2015 Wolfsblvt ( www.pinkes-forum.de )
+ * @license http://opensource.org/licenses/gpl-2.0.php GNU General Public License v2
+ * @author Clemens Husung (Wolfsblvt)
+ */
+
+namespace wolfsblvt\advancedpolls\migrations;
+
+class v1_2_2_data_permissions extends \phpbb\db\migration\migration
```

Comment:
We are still working vor version 1.2.0, so can you rename the file to `v1_2_0_data_permissions`?